tests/_callbacks/test_callback_helpers.py

```diff
@@ -120,23 +117,6 @@ def test_get_callbacks__user_config(tmp_path: Path) -> None:
 
 
 def test_get_checkpoint_model() -> None:
-    model = helpers.get_model()
-    checkpoint_model = callback_helpers.get_checkpoint_model(model=model)
-    assert not isinstance(checkpoint_model, DummyCustomModel)
-
-
-def test_get_checkpoint_model__no_model_getter() -> None:
-    class DummyCustomModel(Module):
-        def feature_dim(self) -> int:
-            return 1
-
-        def forward_features(self, x: Tensor) -> Tensor:
-            return torch.zeros(1)
-
-        def forward_pool(self, x: Tensor) -> Tensor:
-            return torch.zeros(1)
-
     model = DummyCustomModel()
-    checkpoint_model = callback_helpers.get_checkpoint_model(model=model)
-    assert model == checkpoint_model
-    assert isinstance(checkpoint_model, DummyCustomModel)
+    checkpoint_model = model.get_model()
```

Comment:
Can't we remove `test_get_checkpoint_model` completely as `get_checkpoint_model` was removed?